dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -11,9 +11,10 @@
 
```

Comment:
Best to look at this file with the diff settings above set to ignore whitespace (changed python indent)